specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-09-24-preview/desktopvirtualization.json

```diff
@@ -2797,6 +2718,10 @@
           "type": "boolean",
           "description": "Allow a new session."
         },
+        "virtualMachineId": {

```

Comment:
is this a value modifiable by the user, or is it generated by the service/immutable?

Reply:
its immutable adding readonly